# Set your depth image topic here, default is for openni2
depth_topic: "/camera/depth_registered/hw_registered/image_rect"

# Set your face detection topic here
face_topic: "/face_recognizer/faces"

# Set your output topic here
output_topic: "/face_recognizer/faces/realworld/position"

# Set your depth focal length
focal_length: 0.00173667
